README.rst

```diff
@@ -664,21 +674,21 @@ Get notification details
 
 .. code-block:: text
 
-    GET /api/v1/notifications/{pk}/
+    GET /api/v1/notifications/notification/{pk}/
```

Comment:
`/api/v1/notifications/` should become `/api/v1/notification/`

Reply:
But according to following, the convention seems to be
```
/api/v1/<app-name>/<model-name>/{id}
```
![image](https://user-images.githubusercontent.com/32094433/89820552-d385d080-db6a-11ea-9691-e4871b976a28.png)

Reply:
the main point you should notice is that all the URLs use the singular noun of the model, not the plural